src/test/java/seedu/address/logic/commands/FindMeetingCommandTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.CARL;
+import static seedu.address.testutil.TypicalPersons.ELLE;
+import static seedu.address.testutil.TypicalPersons.FIONA;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.MeetingNameContainsKeywordsPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code FindMeetingCommand}.
+ */
+public class FindMeetingCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        MeetingNameContainsKeywordsPredicate firstPredicate =
+                new MeetingNameContainsKeywordsPredicate(Collections.singletonList("first"));
+        MeetingNameContainsKeywordsPredicate secondPredicate =
+                new MeetingNameContainsKeywordsPredicate(Collections.singletonList("second"));
+
+        FindMeetingCommand findFirstCommand = new FindMeetingCommand(firstPredicate);
+        FindMeetingCommand findSecondCommand = new FindMeetingCommand(secondPredicate);
+
+        // same object -> returns true
+        assertTrue(findFirstCommand.equals(findFirstCommand));
+
+        // same values -> returns true
+        FindMeetingCommand findFirstCommandCopy = new FindMeetingCommand(firstPredicate);
+        assertTrue(findFirstCommand.equals(findFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(findFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(findFirstCommand.equals(null));
+
+        // different person -> returns false
+        assertFalse(findFirstCommand.equals(findSecondCommand));
+    }
+
+    @Test
+    public void execute_zeroKeywords_noPersonFound() {
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
+        MeetingNameContainsKeywordsPredicate predicate = preparePredicate(" ");
+        FindMeetingCommand command = new FindMeetingCommand(predicate);
+        expectedModel.updatePersonListFilter(predicate);
+        assertCommandSuccess(command, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getPersonList());
```

Comment:
Would structuring the code here help readability? Referring to [CS2103/T guideline on structuring code logically](https://nus-cs2103-ay2526s1.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically) for context

```suggestion
        MeetingNameContainsKeywordsPredicate predicate = preparePredicate(" ");
        FindMeetingCommand command = new FindMeetingCommand(predicate);
        
        expectedModel.updatePersonListFilter(predicate);
        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
        
        assertCommandSuccess(command, model, expectedMessage, expectedModel);
        assertEquals(Collections.emptyList(), model.getPersonList());
```

Reply:
Will do so thanks